Source/Core/Core/ConfigLoaders/IsSettingSaveable.cpp

```diff
@@ -28,7 +28,7 @@ bool IsSettingSaveable(const Config::Location& config_location)
       return true;
   }
 
-  static constexpr std::array<const Config::Location*, 104> s_setting_saveable = {
+  static constexpr std::array s_setting_saveable = {
```

Comment:
Alternatively, we could use auto for the variable itself (which will be deduced as a std::initializer_list). This avoids having to wait for C++20.

Reply:
Does that actually work?

Reply:
It'll pretty much work the same as a std::array, except with a compiler-generated backing array (the symbol for the array is "reference temporary"). You'll need to use std::begin and std::end instead of the .begin() and .end() member functions, though.